config/v1/types_image_content_policy.go

```diff
@@ -36,11 +36,12 @@ type ImageContentPolicySpec struct {
 	// If the "mirrors" is not specified, the image will continue to be pulled from the specified
 	// repository in the pull spec.
 	//
-	// When multiple policies are defined for the same “source” repository, the sets of defined
+	// When multiple ImageContenPolicy objects are defined for the same “source” repository, the sets of defined
 	// mirrors will be merged together, preserving the relative order of the mirrors, if possible.
 	// For example, if policy A has mirrors `a, b, c` and policy B has mirrors `c, d, e`, the
 	// mirrors will be used in the order `a, b, c, d, e`.  If the orders of mirror entries conflict
 	// (e.g. `a, b` vs. `b, a`) the configuration is not rejected but the resulting order is unspecified.
+	// Define for multiple same “source” repository in one ImageContenPolicy is not allowed.
```

Comment:
Is there a benefit to this restriction? We already have to implement, and define, merging behavior across CRs, so allowing multiple entries within a single CR doesn’t make our job harder at all.

OTOH, sure, the user could in principle always use a single entry per `source`, so this restriction doesn’t remove functionality, either.